pgctl/config.py

```diff
@@ -116,9 +117,12 @@ def from_app(self, path='.'):
             for parentdir in reversed(tuple(search_parent_directories(path)))
         )
 
-    @staticmethod
```

Comment:
Highly doubt it, but nobody is actually using this as a static method currently right?

Reply:
Nope, it's only used below (as an instance method) outside of tests.